### ISSUE TITLE
Accessibility issues and questions

### ISSUE THREAD

**Issue by yarnetsky** [code4lib/2021.code4lib.org#55]
**general**
- header styles very similar in appearance, hard to visually perceive outline of document. h2 is 32px and h3 is 28px. Add additional signifiers for outline (separators or font-weight?) [Ranti]
- skip to content link to bypass navigation (especially on general info pages) [Julia]

**Proposal/Volunteer section**: on both the front and schedule pages
- headers out of order (current order 1 - 4 - 2 - 2 - 4 -2 and should be 1 -2 - 3 - 3 - 2 - 3) [Derek]
- link text "sign up to volunteer" for equal buttons and so it's understandable out of context (links-only scan from screen reader). __label needs to be changed to "Volunteer for Committee"__ [Ranti]

**Keynote page**
- tab order gets caught by pop-up speaker bio section, even when it is hidden (I think section needs aria-hidden=true for when the box is not exposed so the tab order bypasses the link) [Jerry]
- should the speaker cards be a single link? double link for same action. [Jerry]
- No means to toggle or close speaker bio, only option is to open another. [Derek]
- should the panel be captured when it's expanded? [Hold]
- low contrast links in speaker bios (aqua on white 1.9:1) [Derek]

**Accessibility page**
- slide submission email address is an empty link in action 2 [Ranti]
- links under sharing slides should be part of the text, not written out urls (awful to hear on a screen reader) [Ranti]
![Screen Shot 2020-12-07 at 3 43 36 PM](https://user-images.githubusercontent.com/10561752/101403286-033baf80-38a3-11eb-83b1-d059a684c29e.png)

Conduct and Safety page
- `<code>` snippet `/support` is low contrast (coral on dark blue for 2.5:1 ratio). Maybe gold? [Julia]

**Things to check**
- semantic markup -- should sections have id or other identification?
    - i.e. skip to main

**Last review**: Nov. 24-25 including front page, past keynote speakers, and all general info and conduct pages.

**Comment by jscaffrey** [comment]
Re: headers out of order (current order 1 - 4 - 2 - 2 - 4 -2 and should be 1 -2 - 3 - 3 - 2 - 3)
This was an issue that last year's WG struggled with how to improve in 2020 -- the code here differs based on the current state of the conference (pre, peri, post). Maybe we can make more progress on it this year.

**Comment by dzoladz** [comment]
**Speaker Page**
> The other issue was with the speaker page. It sounds like the speaker bios are hidden for sighted users, but are not hidden for screen readers. I get a notification that the link is collapsed, but then I never get a notice that it has been expanded (even after making sure it was expanded), and no matter what I am forced to go through the bio text whether it is visually expanded or not. Looking at the code, it seems the ARIA-expanded is not functioning. I'm not an aria-expert (or even novice, to be honest), but it might be because it uses divs instead of ul? Or it may be because I'm using NVDA, since I know aria doesn't always function correctly with it.

**Comment by dzoladz** [comment]
Individual issues created and assigned
